example/embedded_app_lifecycle/2_c_provider_consumer_app.html

```diff
@@ -13,6 +13,7 @@
       }
       html[hidden] { display: none; }
     </style>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.23.0/polyfill.min.js"></script>
```

Comment:
Why are you adding the poly fill here if it isn’t needed

Reply:
actually polyfill is needed on this page, however, it's not used for XFC but for an explicit Promise call on this page. We call Promise.resolve in secret function. See [here](https://github.com/cerner/xfc/blob/4e524f3b18c0b6f2434db3e4b97a966bb54f1afd/example/embedded_app_lifecycle/2_c_provider_consumer_app.html#L35)